doc/tutorials/open_vswitch.rst

```diff
@@ -0,0 +1,163 @@
+.. _open_vswitch:
+
+Using OVS bridge instead of ACRN bridge
+#######################################
+This tutorial describes how to use Open Virtual Switch
+(OVS) bridge instead of ACRN bridge.
+
+.. note::
+   The OVS is integrated in ClearLinux ``service-os``
+   bundle, and the ClearLinux OS must be at least ``28900`` version.
+
+What is OVS
+***********
+Open vSwitch, abbreviated as OVS, is an open-source implementation of
```

Comment:
@lirui34 I see many comments are marked as resolved, but I don't see changes to the document?  Maybe an updating push wasn't done?

Reply:
Oh, sorry about that, I just requested an internal review in our group yesterday so that they could only see those 'unresolved' comments.
I have pushed an new version just now :smile: